README.rst

```diff
@@ -1,6 +1,17 @@
 barbequeue
 ==========
 
+.. image:: https://badge.fury.io/py/iceqube.svg
+   :target: https://pypi.python.org/pypi/iceqube/
+.. image:: https://travis-ci.org/learningequality/iceqube.svg
+  :target: https://travis-ci.org/learningequality/iceqube
+.. image:: http://codecov.io/github/learningequality/iceqube/coverage.svg?branch=master
+  :target: http://codecov.io/github/learningequality/iceqube?branch=master
+.. image:: https://readthedocs.org/projects/iceqube/badge/?version=latest
+  :target: http://iceqube.readthedocs.org/en/latest/
```

Comment:
looks like some badges got added that don't exist

![image](https://user-images.githubusercontent.com/2367265/33912421-a97f42cc-df4a-11e7-9aa4-fed0b49d7dc0.png)

Reply:
Yeah, I just closed the PR to get something done... will add CodeCov next time around... and docs, maybe too :)

Reply:
https://github.com/learningequality/iceqube/issues/30